src/content/changelog/4-62-0-2020-04-10.md

```diff
@@ -16,14 +16,16 @@ changelog:
 
 - The View Kubernetes API Endpoint ([/lke/clusters/{clusterId}/api-endpoint](/api/v4/lke-clusters-cluster-id-api-endpoint)) endpoint is now deprecated in favor of List Kubernetes API Endpoints ([GET /lke/clusters/{clusterId}/api-endpoints](/api/v4/lke-clusters-cluster-id-api-endpoints)).
 
-- The `version` property of the LKE collection has been updated to be named `k8s_version`. This change affects the following endpoints:
+- The `version` property of an LKE cluster has been updated to be named `k8s_version`. This change affects the following endpoints:
 
     - List Kubernetes Clusters ([GET /lke/clusters](/api/v4/lke-clusters))
     - Create Kubernetes Clusters ([POST /lke/clusters](/api/v4/lke-clusters/#post))
     - View Kubernetes Cluster ([GET /lke/clusters/{clusterId}](/api/v4/lke-clusters-cluster-id))
 
 ### Fixed
 
-- The [LKE collection's](/api/v4/lke-clusters) `label`, `k8s_version`, and `region` properties were incorrectly labeled as being [filterable](/api/v4/#filtering-and-sorting). These properties have been updated in the [Linode API v4 schema](/api/v4) to no longer display the `filterable` label.
+- An [LKE cluster's](/api/v4/lke-clusters) `label`, `k8s_version`, and `region` properties were incorrectly labeled as being [filterable](/api/v4/#filtering-and-sorting). These properties have been updated in the [Linode API v4 schema](/api/v4) to no longer display the `filterable` label.
```

Comment:
Just making this a little more specific that those properties belong to the cluster object